### ISSUE TITLE
Can we make custom markers draggable?

### ISSUE THREAD

**Issue by idrenski** [allenhwkim/angularjs-google-maps#704]
**IMPORTANT**: 

This repository's issues are reserved for feature requests and bug reports. 
Do not submit support questions or request here. Please use stackoverflow instead

**Steps to reproduce and a minimal demo(Plunker Example)**

  - _What steps should we try in your demo to see the problem?_

**Current behavior**
It seems tha custom markers are not draggable. i.e attribute draggable="true" doesn't work.
  - 

**Expected/desired behavior**

Custom markers to be fully functional.
  - 

**Other information**

  -

**Comment by benediktdertinger** [comment]
+1

**Comment by allenhwkim** [comment]
Yes, we can. PR would be appreciated based on the following solution.

http://stackoverflow.com/questions/22127626/can-we-make-custom-overlays-draggable-on-google-maps-v3

**Comment by indianazhao** [comment]
+1

**Comment by softwarenacho** [comment]
+1

I see this example of a working custom draggable marker: http://jsfiddle.net/doktormolle/QRuW8/

I'm trying to implement it over custom-marker.js, but I can't seem to build the directive to be read.
